### DISCUSSION TITLE
rocm 5.5 not getting into extra

### DISCUSSION THREAD

**Discussion by jannesklee** [rocm-arch/rocm-arch#965] [General]
Hi everyone,

I was wondering why version 5.5 is not getting into the extra repository but stays in extra-staging. 

https://archlinux.org/packages/extra-staging/x86_64/rocm-core/

Is there a specific reason why this is the case?

**Comment by tpkessler** [comment]
Hi! This is because pytorch with ROCm doesn't build because of a compiler bug, see https://github.com/ROCm-Developer-Tools/aomp/issues/557
I'm currently working on ROCm 5.6 and hope that the compiler bug is fixed in the latest release.

**Comment by jannesklee** [comment]
That's a pretty specific reason, hopefully it is fixed in 5.6. Thanks for the swift reply and taking care of the package.

**Comment by tpkessler** [comment]
ROCm 5.6 is now in [extra-testing]!
